sigma/plugins/utility/other/define.py

```diff
@@ -0,0 +1,79 @@
+import discord
+import aiohttp
+from bs4 import BeautifulSoup
+
+async def define(cmd, message, args):
+
+    definition = ""
+
+    defNum = 1
+
+    defLast = False
+    
+    hasPrinted = False
+
+    defURL = r"https://www.merriam-webster.com/dictionary/"
+
+
+    if not args:
+        await message.channel.send(cmd.help())
+        return
+    else:
+        try:
+            defNum = int(args[-1])
+            defLast = True
+        except ValueError:
+            pass
+
+        if not defLast:
+            for i in args:
+                defURL += i
+                defURL += "%20"
+        else:
+            argsCopy = args.copy()
+            del(argsCopy[-1])
+            for i in argsCopy:
+                defURL += i
+                defURL += "%20"
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(defURL) as resp:
+            rawDef = await resp.text()
+    
+    try:
+        rawDef = rawDef[rawDef.index("<!--JC: actual definitions-->")-10:]
+    except:
+        embed = discord.Embed(title=":unamused: An error has occured. Silly you! Enter a real word next time!", color=0xFF8EE6)
+        await message.channel.send(None, embed=embed)
+        cmd.log("The user did not input a real word. Either that or the Merriam-Webster HTML files were changed, and the whole command needs to be reworked.")
+
+    rawDefArr = rawDef.split("<!--JC: actual definitions-->")
+
+    try:
+        soupObj = BeautifulSoup(rawDefArr[defNum], "html.parser")
+    except:
+        embed = discord.Embed(title=":exclamation: An error has occured. This number definition does not exist; please try again with a different number.", color=0xFF0000)
+        await message.channel.send(None, embed=embed)
+        cmd.log("This error should never trigger, but if it does, something is wrong.")
```

Comment:
You can remove the cmd.log here. It tells us nothing more than what we already know (the discord message posting that the definition isn't found). cmd.log is used for reporting errors without printing how badly you goofed up in the discord chat.